api/camunda/camunda-openapi.yaml

```
@@ -731,52 +837,67 @@ paths:
           description: >
             The decision requirements search query failed.
             More details are provided in the response body.
-  /authorizations:
-    post:
+
+  /authorizations/{ownerKey}:
+    patch:
       tags:
         - Authorization
-      summary: Create an authorization
-      operationId: createAuthorization
+      summary: Patch an authorization
+      description: Manage the permissions assigned to the authorization
```

```suggestion
      description: Manage the permissions assigned to the authorization.
```

category: casing

owner: me

---

https://github.com/camunda/camunda/pull/22101